core/src/main/java/org/elasticsearch/action/support/WriteRequest.java

```diff
@@ -65,45 +65,70 @@ default R setRefreshPolicy(String refreshPolicy) {
         /**
          * Don't refresh after this request. The default.
          */
-        NONE,
+        NONE((byte) 0, Boolean.FALSE.toString()),
```

Comment:
nit: I am wondering if using "true" and "false" would be more readable, simply because I always forget if it's lowercase or uppercase.